package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Browser Performance regression suite",
   "main": "lib/index.js",
   "scripts": {
-    "test": "grunt test",
+    "test": "bower install grunt test",
```

Comment:
shoult this be `bower install && grunt test` ? So that it works on Windows also ?